client/wfnews-war/src/main/angular/src/app/components/active-wildfire-map/active-wildfire-map.component.scss

```diff
@@ -437,7 +437,7 @@ mat-panel-title {
     }
 
     .map {
-      height: calc(100% - 128px) !important;
+      height: 90vh !important;
```

Comment:
Is this 90vh reliable all the time? If the footer isn't set to 10vh, use a calc from 100 instead